workers/unity/Packages/io.improbable.gdk.mobile/Editor/iOSUtils.cs

```diff
@@ -80,37 +81,43 @@ public static (List<DeviceLaunchConfig> emulators, List<DeviceLaunchConfig> devi
             return (availableSimulators, availableDevices);
         }
 
-        public static void Build(string developmentTeamId)
+        public static void MenuBuild(string developmentTeamId)
         {
             try
             {
                 EditorUtility.DisplayProgressBar("Preparing your Mobile Client", "Building your XCode project", 0f);
-
-                if (!Directory.Exists(XCodeProjectPath))
-                {
-                    Debug.LogError("Unable to find an XCode project. Did you build your iOS worker?");
-                    return;
-                }
-
-                if (string.IsNullOrEmpty(developmentTeamId))
-                {
-                    Debug.LogError("Development Team Id was not specified. Unable to build the XCode project.");
-                    return;
-                }
-
-                if (!TryBuildXCodeProject(developmentTeamId, out var xcBuildErrors))
-                {
-                    Debug.LogError("Failed to build your XCode project. " +
-                        "Make sure you have the Command line tools for XCode (https://developer.apple.com/download/more/) " +
-                        $"installed and check the logs:\n{string.Join("\n", xcBuildErrors)}");
-                }
+                Build(developmentTeamId);
             }
             finally
             {
                 EditorUtility.ClearProgressBar();
             }
         }
 
+        public static void Build(string developmentTeamId = "")
```

Comment:
Can we use `string.Empty` or is that static, not const? I can't recall

Reply:
We cant use it as its not const